times_reader/datatypes.py

```diff
@@ -262,6 +262,8 @@ def _read_veda_tags_info(veda_tags_file: str) -> Dict[Tag, Dict[str, str]]:
             if "tag_fields" in tag_info:
                 # The file stores the tag name in lowercase, and without the ~
                 tag_name = "~" + tag_info["tag_name"].upper()
+                if not Tag.has_tag(tag_name):
```

Comment:
@siddharth-krishna I am thinking of `veda_tags.json` as of reference which may have more information that is needed / used by the tool. In this case I am wondering whether the direction of the warning should change? I.e. the tool should alert if a tag is defined, but the info about it is not present in `veda_tags.json`?